content/docs/installation/scala.md

```diff
@@ -1,11 +1,51 @@
 ---
 title: Cucumber-Scala
 subtitle: Scala
-img: /img/scala.png
+svg: installation/scala.svg
 implementation: unmaintained
 weight: 1315
 ---
 
+----
+
 **This package is currently in need of new maintainers.**
 
 Please see [Cucumber-Scala](https://github.com/cucumber/cucumber-jvm-scala).
+
+----
+
+Cucumber-Scala is published in the central Maven repository.
+You can install it by adding dependencies to your project.
+
+{{% note "Dependencies"%}}
+Make sure the Cucumber version is the same for all Cucumber dependencies.
+{{% /note %}}
+
+# Maven
+
+```xml
+<dependency>
+    <groupId>io.cucumber</groupId>
+    <artifactId>cucumber-scala_2.13</artifactId>
+    <version>{{% version "cucumberjvm" %}}</version>
```

Comment:
Will the scala version always match the "cucumberjvm" version? If not, I suggest adding its own version number. In versions.yaml I think.

Reply:
There's a small chance/risk that version will not always match indeed!